docs/content/en/blog/news/control-plane-on-ecs.md

```diff
@@ -0,0 +1,283 @@
+---
+date: 2022-02-07
+title: "PipeCD best practice 02 - control plane on ECS"
+linkTitle: "PipeCD best practice 02"
+weight: 996
+description: "This blog is a part of PipeCD best practice series, a guideline for you to operate your own PipeCD on Amazon ECS."
+author: Yohei Namba ([@kevin55156](https://twitter.com/kevin55156))
+---
+
+This blog is a part of PipeCD best practice series, a guideline for you to operate your own PipeCD.
+Currently, we can deploy the control plane to kubernetes cluster, but some developers that would like to introduce PipeCD can not prepare kubernetes environments. PipeCD started supporting deployment control plane on docker-compose, so this blog introduce one way how to deploy control plane on ECS.
+
+### Architecture
+> Note: Please refer to [architecture-overview](docs/user-guide/managing-controlplane/architecture-overview/) docs for definitions of PipeCD components such as server, ops, cache, datastore and filestore.
+
+![](/images/control-plane-on-ecs.png)
+
+### RDS(datastore)
+It is possible to use RDS as a datastore. Edit your configuration file for the control plane according to your RDS setting.
+```yaml=
+  datastore:
+    type: MYSQL
+    config: 
+        url: root:password@tcp(arn_of_rds:3306) 
+        database: quickstart
+```
+
+### Redis(cache)
+It is possible to use redis as a cache. Note the endpoint of redis for the task definition.
+
+
+### S3(filestore)
+It is possible to use S3 as a filestore. The filestore contains state files that describe your secure infrastructure, so make sure to make the bucket private. Only allow pipecd-server(ECS) to access this bucket..
+
+### Secrets Manager
```

Comment:
How about moving this session above RDS and showing an example of what is a control-plane-config.yaml/envoy-config.yaml and how it looks like? I mean you can also link to GitHub URL where we stored those files in the example code.